src/assets/app.extensions.json

```diff
@@ -1219,6 +1220,201 @@
               }
             ]
           },
+          {
```

Comment:
i think it's better to add id's for each scenario fur future reusabillity / testing purposes.

Reply:
Yes, good catch. I forgot to come back to do that.

Reply:
Done. Please re-review, thanks.